src/benchmarks/benchmarks.cpp

```diff
@@ -297,6 +297,22 @@ int main(int argc, char* argv[]) // NOLINT
                     | rxcpp::operators::subscribe<int>([](int v){ ankerl::nanobench::doNotOptimizeAway(v); });
             });
         }
+        SECTION("create+flat_map(just(v*2))+subscribe")
+        {
+            TEST_RPP([&]()
+            {
+                rpp::source::create<int>([](const auto& obs){ obs.on_next(1); })
+                    | rpp::operators::flat_map([](int v) { return rpp::source::just(v * 2); })
```

Comment:
Could you use “create” there instead of just to measure there only “flat_map”, not “just” :)

Reply:
done